zos_subsystems/cics/cmci/module_defaults/README.md

```diff
@@ -0,0 +1,93 @@
+# Using module_defaults to install a program definition
+
+This sample playbook demonstrates how to use the `cmci_get` module from the
+`ibm_zos_cics` collection to retrieve operational data from running CICS
+regions.
```

Comment:
You're using all the `cmci_` modules in the playbook to lifecycle a program resource right, rather than just using `cmci_get` to retrieve data.